docs/ingress-tutorial.md

```diff
@@ -85,10 +85,16 @@ The server process runs as `root` so that it can open any privileged ports the c
 
 Get the client `Deployment` manifest and edit it:
 
+for kubernetes =< v1.15
```

Comment:
That's not strictly true, apps/v1 has been around since I think 1.13 or earlier.